cluster-install/readme.adoc

```diff
@@ -89,6 +89,8 @@ When setting up a cluster you have two options on how the nodes in the cluster c
 
 You'll need to choose one of the two options. Instructions for both options are provided below, and the examples in the workshop should work with either option. Creating a gossip-based cluster requires less setup and will be used in this workshop, unless otherwise specified.
 
+NOTE: If you're using London region (eu-west-2) you need to add `--cloud aws` as kops is not identifying that eu-west-2 indicates AWS. Further info https://github.com/kubernetes/kops/issues/1267
```

Comment:
Change both occurrences of eu-west-2 to `eu-west-2`?

Reply:
Done